### PR TITLE
Move Webui::Users::Tasks bootstrap views to Webui namespace.

### PR DESCRIPTION
**Pull request by vpereira**
Move the user tasks to Bootstrap.

To check the view, click on "Tasks" on the menu, beside the link to your profile in the main page. There you are going to see requests and other pending tasks for your user.

### CODE REVIEW

File: src/api/app/views/webui/users/tasks/index.html.haml [hunk 1]
```diff
@@ -1,85 +1,78 @@
-- @pagetitle = "Tasks for #{User.session!}"
-- @layouttype = 'custom'
-- @crumb_list = [@pagetitle]
+:ruby
+  @pagetitle = "Tasks for #{User.session!}"
+  @crumb_list = [@pagetitle]
```
Comment (DavidKang) [inline src/api/app/views/webui/users/tasks/index.html.haml]: `@crumb_list` is only for the old UI, so this should be needed

File: src/api/app/views/webui/users/tasks/index.html.haml [hunk 2]
```diff
@@ -1,85 +1,78 @@
...
-                %td= link_to(shortened_prj, project_show_path(pi[:package][:project]))
-                %td= link_to(shortened_pkg, package_show_path(pi[:package][:project], pi[:package][:name]))
-                %td
-                  - pi[:issues].each do |i|
-                    = link_to i[:label], i[:url], title: i[:summary]
-                    ,
-                %td= link_to(sprite_tag('monitor', title: 'Monitor'), project_monitor_path(pi[:package][:project], pkgname: pi[:package][:name]))
+  .card-body
+    .tab-content#reviews-in
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: user_requests_path(User.session!),
```
Comment (DavidKang) [inline src/api/app/views/webui/users/tasks/index.html.haml]: Please, use the partials that are into `webui/shared`

File: src/api/app/views/webui/users/tasks/index.html.haml [hunk 3]
```diff
@@ -1,85 +1,78 @@
...
+          All Requests
+          %span.badge.badge-primary
+            = User.session!.requests.count
+      %li.nav-item.dropdown
+        %a.nav-link.dropdown-toggle{ href: '#', 'data-toggle': 'dropdown', 'role': 'button', 'aria-expanded': 'false', 'aria-haspopup': 'true' }
+        .dropdown-menu.dropdown-menu-right
+  .card-body
+    .tab-content
+      .tab-pane.fade.show.active#requests-in{ role: 'tabpanel', 'aria-labelledby': 'requests-in-tab' }
+        = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: user_requests_path(User.session!),
```
Comment (DavidKang) [inline src/api/app/views/webui/users/tasks/index.html.haml]: Please, use the partials that are into `webui/shared`

Comment (vpereira) [reply src/api/app/views/webui/users/tasks/index.html.haml]: done

File: src/api/app/views/webui/users/tasks/index.html.haml (hunk 2 above)
Comment (vpereira) [reply src/api/app/views/webui/users/tasks/index.html.haml]: done

File: src/api/app/views/webui/users/tasks/index.html.haml (hunk 1 above)
Comment (vpereira) [reply src/api/app/views/webui/users/tasks/index.html.haml]: removed everything since in this page we don't need `@pagetitle` too

Comment (dmarcoux) [reply src/api/app/views/webui/users/tasks/index.html.haml]: `@pagetitle` is [displayed in the layout](https://github.com/openSUSE/open-build-service/blob/e9a4d1f8cb8cfb047572d425ab2cd57660548630/src/api/app/views/layouts/webui/webui.html.haml#L10), so we should definitely keep it.

Comment (vpereira) [reply src/api/app/views/webui/users/tasks/index.html.haml]: it is displayed *if* there. I don't think that I need to repeat the Breadcrumb there. What do you think @dmarcoux ?

Screenshot_2019-09-16_11-06-15

File: src/api/app/views/webui/users/tasks/index.html.haml (hunk 2 above)
Comment (dmarcoux) [reply src/api/app/views/webui/users/tasks/index.html.haml]: This one still needs to be changed.

Comment (vpereira) [reply src/api/app/views/webui/users/tasks/index.html.haml]: done!

**Pull request merged**